src/en/space-station-14/round-flow/proposals/ninja.md

```diff
@@ -0,0 +1,94 @@
+# Space Ninja V2
+
+| Designers | Implemented | GitHub Links |
+|---|---|---|
+| Warpzoned, Doru991 | :warning: Partially | TBD |
+
+## Overview
+
+This proposal is aimed towards addressing some issues that Space Ninjas currently possess, as well as adding some extra flavour and other miscellaneous content.</p>
+<p>New abilities have been added, and existing ones have been tailored to better fit with the stealth-based gameplay the Spider Clan so endorses. The Space Ninja now also come with an extra assortment of apparel, both useful and cool-looking.</p>
+<p>The Space Ninja's weapons have gotten a slight nerf, to compensate, new weapons and armament interactions have been added.
+
+## Background
+
+Currently, albeit being an extremely fun mid-round antag role and definitely one of the better ones, Space Ninjas aren't correctly geared towards what they should be the best at: Stealthing - Which is what this document hopes to address, mostly to add some more uniqueness to antagonist roles since a lot of them can be played by just killing everything in your way, and this includes Space Ninjas.
+
+## Spider Clan Code
+
+All Space Ninjas will now have a Spider Clan Code they will have to live by, and this thus affects gameplay substantially by imposing certain restrictions.
+
+- The already existing one, being the inability to shoot ranged weapons, will be included in this Code.
```

Comment:
Ninjas should get a roundstart Scroll of the Spider Clan Code, outlining an "honor code" that promotes treating enemies with respect and similar things (not RRing them for example). New players playing ninja might change their play style to fit this honor code given they are playing a ninja character, reducing murderboning. Obviously someone can ignore this, but I think it would immerse people in the role of a ninja more.

Reply:
> Ninjas should get a roundstart Scroll of the Spider Clan Code, outlining an "honor code" that promotes treating enemies with respect and similar things (not RRing them for example). New players playing ninja might change their play style to fit this honor code given they are playing a ninja character, reducing murderboning. Obviously someone can ignore this, but I think it would immerse people in the role of a ninja more.

I was discussing the honor code mechanics with chief funk officer the other day, and it eventually devolved into a pretty funny idea that was as follows: basically, if the ninja kills too many personnel, eventually there'll be a "Son of X (killed victim)" that's tasked with finding and killing the ninja at any costs, a sort of terminator, and if said "Son of X" is killed there'll be a "Son of Son of X", so on and so forth.

Personally, I found this hilarious and would love it or something similar to be added, however, I think a more appropriate approach was for there to be a DeathSquad equivalent in the Spider Clan that if the ninja kills too many people one spawns in as a ghost raffle, and they have one and only objective: to kill the ninja, as they have tainted their code of honour, afterwards, they teleport back to base (acidify).